app/models/dimensions/organisation.rb

```diff
@@ -0,0 +1,8 @@
+class Dimensions::Organisation < ApplicationRecord
+  validates :title, presence: true
+  validates :slug, presence: true
+  validates :link, presence: true
+  validates :content_id, presence: true
```

Comment:
I think you meant to write `organisation_id` here?

Reply:
This may also be redundant or need a better name as we already have an implicit `id` attribute. If we were to drop the `organisation_` suffix how would this value differ?